src/com/epam/javatrainings/thread/Main.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.javatrainings.thread;
+
+import java.util.Scanner;
+
+public class Main {
+  public static void main(String[] args) {
+    Scanner scan = new Scanner(System.in);
+    System.out.println("How much time (In Minutes) ?");
+    int timet= scan.nextInt() * 60; // Convert to seconds
+    int sec=scan.nextInt();
```

Comment:
The second input needs to have action message as well

Reply:
done